.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+.idea/
```

Comment:
@develar Could you drop that last commit? Please add `.idea` to your global gitignore.